code/modules/clothing/suits/bio.dm

```diff
@@ -36,14 +36,14 @@
 	strip_delay = 70
 	put_on_delay = 70
 	burn_state = FIRE_PROOF
-
+	hide_tail

```

Comment:
Sorry! I should've been more specific and less incorrect. `HIDETAIL` goes in `flags_inv` and examples can be seen [here](https://github.com/ParadiseSS13/Paradise/search?q=HIDETAIL&unscoped_q=HIDETAIL)

Reply:
Oh! It's... already in there. That makes things extra simple.